data-reconciliation-app/test/unit-test/repositories/kv-repository.test.ts

```diff
@@ -0,0 +1,60 @@
+import * as polyfill from "@microsoft/ccf-app/polyfill.js";
+
+import * as jscrypto from "crypto";
+import {
+  IKeyValueRepository,
+  KeyValueRepository,
+} from "../../../src/repositories/kv-repository";
+import { DataRecord } from "../../../src/models/data-record";
+import { describe, expect, test, beforeEach, afterEach } from "@jest/globals";
+
+describe("Key value pair Repository", () => {
```

Comment:
Our repository implementation is a wrapper/abstraction on `kyStore`. By using `jest.Spy`, we could avoid the `polyfill`?

Reply:
There is an issue when you call any function related to ccf-kvstore and it throws an exception. so the ccf-kvstore need to be mocked

```
Test suite failed to run > Jest encountered an unexpected token
/workspaces/aym-ccf-app-samples/data-reconciliation-app/node_modules/@microsoft/ccf-app/index.js:16
    export * from "./kv.js";

    SyntaxError: Unexpected token 'export'
```